### PR TITLE
Fixed overflow link from Bio-card

### PR DESCRIPTION
**Pull request by Jolah1**
On "people's page" The affiliation has been replaced with the university name as it should be instead of the overflowing URL as shown in Issue #495 
link

University name was gotten from the URL link.

### CODE REVIEW

**Comment by Jolah1** [comment]
@bebatut please kindly review.

The initial PR conflict was caused by the auto formatter on my code editor.

File: _data/people.yaml [hunk 1]
```diff
@@ -5773,7 +5773,7 @@ sk-sahu:
   pronouns: he/him
   twitter: sangram_ksahu
 sladjana-lukic:
-  affiliation: Https://Www.Adelphi.Edu/Faculty/Profiles/Profile.Php?Pid=0966
+  affiliation: Adelphi University, New York
```
Comment (bebatut) [inline _data/people.yaml:5776]: Can you remove New York? That could go to the city information

**Comment by Jolah1** [comment]
Alright….on it…

File: _data/people.yaml (hunk 1 above)
Comment (Jolah1) [reply _data/people.yaml]: > Can you remove New York? That could go to the city information

Done @bebatut

**Review by AndreaSanchezTapia — APPROVED** [review]

**Pull request merged**